src/main/kotlin/com/team4099/robot2023/subsystems/groundintake/GroundIntake.kt

```diff
@@ -283,7 +281,7 @@ class GroundIntake(private val io: GroundIntakeIO) : SubsystemBase() {
 
   fun intakeCommand(): CommandBase {
     val returnCommand = runOnce {
-      requestedState = GroundIntakeConstants.ArmStates.TARGETING_POSITION
+      requestedState = GroundIntakeRequest.TargetingPosition(intakeAngle.get())
       armPositionTarget = intakeAngle.get()
```

Comment:
you don't need this `armPositionTarget` thing anymore because you build that into the request